Source/Extensions/Blazorise.SignaturePad/SignaturePad.razor.cs

```diff
@@ -131,6 +131,18 @@ public async Task Clear()
         }
     }
 
+    /// <summary>
+    /// Undos the last stroke if there is any.
+    /// </summary>
+    /// <returns>A task that represents the asynchronous operation.</returns>
+    public async Task Undo()
+    {
+        if ( JSModule is not null )
+        {
+            await JSModule.Undo( ElementRef, ElementId );
```

Comment:
Does `Undo` need to do more stuff? Like triggering `ValueChanged` with the new `byte[]`?

Reply:
Done